src/object/tilemap.cpp

```diff
@@ -438,9 +438,18 @@ TileMap::draw(DrawingContext& context)
 {
   // skip draw if current opacity is 0.0
   if (m_current_alpha == 0.0f) return;
-
+  
   context.push_transform();
 
+  const bool normal_speed = m_editor_active && Editor::is_active();
+  float speed_x = normal_speed ? 1.0f : m_speed_x;
+  float speed_y = normal_speed ? 1.0f : m_speed_y;
```

Comment:
These probably could be marked as `const`.

Reply:
I fixed both, thanks ^.^